.github/workflows/test.yml

```diff
@@ -19,6 +19,10 @@ jobs:
             python-version: 3.7
           - os: ubuntu-latest
             python-version: 3.8
+          - os: macos-latest
+            python-version: 3.7
+          - os: macos-latest
+            python-version: 3.8
```

Comment:
I would stick to our conservative approach and test macOS only for Python 3.6 as we do in `audformat`:

```yaml
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ ubuntu-18.04, windows-latest, macOS-latest ]
        python-version: [3.6]
        include:
          - os: ubuntu-latest
            python-version: 3.7 
          - os: ubuntu-latest
            python-version: 3.8
```

Reply:
OK sure, I changed it: 96d8ba1e9b0e025ed03c01eadbc946a398c1498b